app/observe-server-gn-test/src/main/resources/conf/app.conf

```diff
@@ -0,0 +1,46 @@
+#
+# Observe server configuration for the testing server
+#
+
+# mode for the test server is production but we point to the test odb
+mode = production
+site = GN
+
+smart-gcal {
+  # Location of the csv files
+  smart-gcal-dir = "/home/software/.observe/smartgcal"
+}
+
+# Single sign-on configuration
+lucuma-sso {
+  # THESE VALUES DON'T WORK FOR SOME REASON
+  #service-token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzUxMiJ9.eyJpc3MiOiJsdWN1bWEtc3NvIiwic3ViIjoiMjU2IiwiYXVkIjoibHVjdW1hIiwiZXhwIjoyMzMyMDExNTI0LCJuYmYiOjE3MDEyOTE1MTQsImlhdCI6MTcwMTI5MTUxNCwKICAibHVjdW1hLXVzZXIiIDogewogICAgInR5cGUiIDogInNlcnZpY2UiLAogICAgImlkIiA6ICJ1LTEwMCIsCiAgICAibmFtZSIgOiAib2JzZXJ2ZS1nbi10ZXN0aW5nIgogIH0KfQ.h3Vk5jwOqZ4phA-1xmOLN_CHN4oUaQM8OY1ZrU65PUAOv1L5bA7-0W4i_Z1LonGZc4uOoVGJozyclsHYZNksLva9JRCOzFEjDvM-_kUSkAtxRF3K7uQTzmT1UCj0iEe_iLkFdlNilkQjK6yo5isL6uk_qcwmaXFrK03SAbMe7QR0_uvBonu-OBKho1kRCZO1r4wTF4lW3MDcFC1KseG43Xx58UDX-lktC3-oJqv37vdZRi1Uyz9DLCW9hCO_Y6v_QhHg44byYMP4fAlRQ_RMrNPaXES8Wwp3O7wVChi5ceiMc2Humd1-BTT9jsUg-wpnbKr7FLAcMtwDS6MntQDhxSPeDqkstlb-nfG6yOWy91YhcrrSFphaaUiwrxgBWH1qDMSUg_uLG_BZuC6Q54epzf1A-A-hFhURGWhdmRGFtRgkMbv_zU1VMJrftW6d0TKg_pG3MG0OXgeZHwWZQ4dtkWYyzm14J9UJmGFv-i-jWGtep37bXeWQuvyiqwacfhszAd2Ph5G6hQ94xWTwkTsjYxGEjoYXwdranxSXWZhhkNUroCNwprgmp00m88BBL7YMD_o7K1JMYefY9_7HCDCmJ_uicFRukPIL-B1AuIMnTmxyxAMeivraKXYQYNLciFmkkkupOu31dXZhkgbOGj_QUXp7Z5hMwypiMyylt4jX2yo"
```

Comment:
Are we exposing tokens in github?

Reply:
🤦 They are already exposed, this is ODB's production token, since I couldn't get a new token to work. I'll change it tomorrow.